test/pylib/manager_client.py

```diff
@@ -158,3 +158,14 @@ async def start_stopped(self) -> None:
         """Start all previously stopped servers"""
         await self._get(f"/cluster/start_stopped")
         self._driver_update()
+
+    async def server_get_config(self, server_id: str) -> dict[str, object]:
+        resp = await self._get(f"/cluster/server/{server_id}/get_config")
+        if resp.status != 200:
+            raise Exception(await resp.text())
+        return await resp.json()
```

Comment:
I'm surprised that the HTTP library has a json decoding function, which has nothing to do with JSON :-( But ok...